packages/stores/src/queries/pool-incentives/incentivized-pools.ts

```diff
@@ -118,6 +118,52 @@ export class ObservableQueryIncentivizedPools extends ObservableChainQuery<Incen
     }
   );
 
+  readonly computeAPYWithExternalIncentives = computedFn(
```

Comment:
As Aidan and I agreed in the meeting, IMO we should calculate APR for a duration for external gauges separately from internal gauges.

Reply:
Then, we add them in the view layer.

This way, we would be able to separately display the APR for just external gauges of a certain duration.

Reply:
Yeah we can use that code snippet for our current needs, but we'd also have those bullet points available as an option.

Reply:
Reminder: we're currently redesigning the pool detail page in a major way, so flexibility in our APIs is extra critical.

I know for a fact we will be displaying external incentive APRs in a different way.